ee/session_recordings/playlist_counters/recordings_that_match_playlist_filters.py

```diff
@@ -164,10 +164,6 @@ def count_recordings_that_match_playlist_filters(playlist_id: int) -> None:
         REPLAY_TEAM_PLAYLIST_COUNT_FAILED.inc()
 
 
-@shared_task(
-    ignore_result=True,
-    queue=CeleryQueue.SESSION_REPLAY_PERSISTENCE.value,
-)
 def enqueue_recordings_that_match_playlist_filters() -> None:
```

Comment:
this doesn't need to be a task (and other similar tasks don't have this method as a task)